xikolo-ios/ViewControllers/Courses/VideoViewController.swift

```diff
@@ -32,16 +36,28 @@ class VideoViewController : UIViewController {
             self.video = videoComplete
             self.summaryView.text = videoComplete.summary
             self.performSegueWithIdentifier("EmbedAVPlayer", sender: self.video)
+            self.openSlidesButton.hidden = self.video?.slides_url == nil
         }
     }
 
     override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
-        let destination = segue.destinationViewController as! AVPlayerViewController
-        let myVideo = sender as! Video
-        if let urlString = myVideo.single_stream_hls_url {
-            let url = NSURL(string: urlString)
-            destination.player = AVPlayer(URL: url!)
+        switch segue.identifier {
+            case "EmbedAVPlayer"?:
+                let destination = segue.destinationViewController as! AVPlayerViewController
+                let myVideo = sender as! Video
+                if let urlString = myVideo.single_stream_hls_url {
+                    let url = NSURL(string: urlString)
+                    destination.player = AVPlayer(URL: url!)
+                }
+            case "ShowSlides"?:
+                let vc = segue.destinationViewController as! WebViewController
+                if let url = (sender as! Video).slides_url {
+                    vc.url = Brand.BaseURL + url
```

Comment:
Please modify the model to use a `URLAttribute` instead. See `CourseSpine` for how to do it. That way you don't have to add the BaseURL here anymore.